### ISSUE TITLE
Feature Request: Support for ANF volume from snapshot - azurerm_netapp_volume 

### ISSUE THREAD

**Issue by razor5678** [terraform-providers/terraform-provider-azurerm#9954]
We can create a new Azure NetApp Files volume, however, I don't see an option to create a new volume from a snapshot / in-place snapshot revert.

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s)

* azurerm_netapp_volume 

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/netapp_volume
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/netapp_snapshot
https://docs.microsoft.com/en-us/azure/azure-netapp-files/azure-netapp-files-manage-snapshots

**Comment by paulomarquesc** [comment]
This was addressed in PR https://github.com/terraform-providers/terraform-provider-azurerm/pull/10906 released on version 2.51.0.

**Comment by katbyte** [comment]
closing as per @paulomarquesc's request
